compiler/src/sct_checker_forward.ml

```diff
@@ -1330,7 +1327,11 @@ let init_constraint fenv f =
   let process_return i x annot =
     let loc = L.loc x and x = L.unloc x in
     let an = Option.bind sig_annot (SecurityAnnotations.get_nth_result i) in
-    let ls, _ = parse_var_annot ~kind_allowed:false ~msf:(not export) annot in
+    let ls =
+      match parse_var_annot ~kind_allowed:false ~msf:(not export) annot with
+      | exception Annot.AnnotationError (loc, _msg) ->
+         warn ~loc "ignored annotation"; []
```

Comment:
You don't want to add msg in the warning :-)

Reply:
Of course not.

Reply:
I’ve removed all of this: the SCT checker no longer attempt to parse `#poly` annotations.